src/runner/browser-env/vite/browser-modules/mocha/index.ts

```diff
@@ -62,7 +62,8 @@ export class MochaWrapper {
             let error: Error | undefined = undefined;
 
             try {
-                await (runnableToRun.fn as Mocha.AsyncFunc)?.call({} as unknown as Mocha.Context);
+                const ctx = { browser: window.browser };
+                await (runnableToRun.fn as unknown as RunnableFn).call(ctx, ctx);
```

Comment:
Call runnable with `browser` as `this` and as first argument of runnable callback. I will add `currentTest` in one of the following PR